### ISSUE TITLE
MQTT Broker username and password

### ISSUE THREAD

**Issue by baahver** [moroen/ZWave-MQTT-plugin#15]
Am I right that there is no option to enter the Mosquitto/MQTT broker username and password in the plugin?
The Zigbee2MQTT has this option.

Edit: I do not understand python, I am not an coder/developer. 
In Zigbee2Mqtt I see that it is easy to add an username and password field to the plugin, but then I do not see how this username and password can be used to connect to the mosquitto/mqtt server. 

Some research as a n00b in python has learned me that domoticz 'library' or 'module' (import domoticz And from api.connection import subscribe_topics, connect_to_broker) is used to connect to Mqtt server. I can not find much information how username and password are used then. In most examples in python where username and password are used to secure connect to MQTT broker, Paho-mqtt is used. Is it possible or undesirable to convert to paho?

**Comment by heggink** [comment]
From what I saw, just adding username and password to the plugin is enough for the plugin to add the authentication itself. It's built-in if the fields are defined.
If you add the 2 lines that I added in the PR to your plugin.py file then you can chek workings already until this gets merged.

**Comment by baahver** [comment]
[domoticz-zwave2mqtt.log](https://github.com/moroen/ZWave-MQTT-plugin/files/6960799/domoticz-zwave2mqtt.log)
Hi Heggink, Thank you for your respons. 
I just added both lines. Now I can enter the MQTT username and password. 
In Domoticz log, all error disappear. So this looks good. 
But No devices show up. I look for zwave and mqtt, but nothing. I pushed the save settings in ZwveJS2MQTT webpage, what should trigger the push off devices according to the wiki. But still no devices in Domoticz. 

Edit:
with those 2 lines the error disappears and the connection is made. I had to restart domoticz and did a restart of docker.
I also had to set the 192.168.x.x lan ip-address in ZwaveJS2MQTT instead of localhost or 127.0.0.1. This is because docker can by default not access host computer direct. 
Devices do appear in Domoticz now.
Thank you Heggink and Moroen! I will experiment further.
